samples/bluetooth/mtu_update/README.rst

```diff
@@ -0,0 +1,67 @@
+.. _bluetooth_mtu_update_sample:
+
+Bluetooth: MTU Update
+#####################
+
+Q&A:
+****
+
+  Question: What do I have to do to update my ATT MTU?
+
+| Answer: To get an ATT MTU of x octets, do the following:
+| Set :kconfig:option:`CONFIG_BT_L2CAP_TX_MTU` to at least x
+| Set :kconfig:option:`CONFIG_BT_BUF_ACL_RX_SIZE` to at least x + L2CAP header
```

Comment:
What is the actual number for the size of the L2CAP header? I think we should have the number here.